app/main/util/sendgrid.py

```diff
@@ -0,0 +1,39 @@
+from app.main.config import sendgrid_key, from_mail
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+from logging import getLogger
+from process import make_async
+
+
+LOG = getLogger(__name__)
+
+
+@make_async
+def send_mail(to_mail, mail_subject, mail_body):
```

Comment:
Create a different function which calls this in async: https://github.com/thealphadollar/TriNetra-Clone/blob/35f9de734052d0a0de232e13434ef7ca01a05663/app/main/service/train.py#L20